preview env: create two workspace classes default and small

jenting opened the pull request:
## Description
<!-- Describe your changes in detail -->
Create two workspace classes in the preview environment so we can test workspace classes in the preview environment.
- Default workspace class with 30Gi disk
- Small workspace class with 20Gi disk

<img width="693" alt="image" src="https://user-images.githubusercontent.com/49380831/192561689-bd20570b-b6db-49cd-972f-7d4e31443655.png">

## Related Issue(s)
<!-- List the issue(s) this PR solves -->
Related to #12638

## How to test
<!-- Provide steps to test this PR -->
- Open the workspace preview.
- Enable the feature flag PVC.
- Open the workspace, and check the PVC size is 30Gi.
- Change your workspace class preference to 20Gi
- Open a new workspace, and check the PVC size is 20Gi.

## Release Notes
<!--
  Add entries for the CHANGELOG.md or "NONE" if there aren't any user facing changes.
  Each line becomes a separate entry.
  Format: [!<optional for breaking>] <description>
  Example: !basic auth is no longer supported
  See https://www.notion.so/gitpod/Release-Notes-513a74fdd23b4cb1b3b3aefb1d34a3e0
-->
```release-note
None
```

## Documentation
<!--
Does this PR require updates to the documentation at www.gitpod.io/docs?
* Yes
  * 1. Please create a docs issue: https://github.com/gitpod-io/website/issues/new?labels=documentation&template=DOCS-NEW-FEATURE.yml&title=%5BDocs+-+New+Feature%5D%3A+%3Cyour+feature+name+here%3E
  * 2. Paste the link to the docs issue below this comment
* No
  * Are you sure? If so, nothing to do here.
-->
None

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

jenting commented:
/werft run

jenting commented:
We need to consider extending the default disk from 30Gi to 50Gi https://github.com/gitpod-io/gitpod/blob/1b995ef47f6631c09c286103db281912d724fd1d/.werft/vm/manifests.ts#L42
Otherwise, if the developer uses the disk over 30Gi, no more data could be written into the disk.
But it also has a side effect that all the default disk sizes will increase to 50Gi.

Or, we could create a small workspace class that is smaller than 30Gi disk, for example, 20Gi, since it's for testing purposes.

WDYT? @gitpod-io/platform

jenting commented:
/werft run

mads-hartmann reviewed (approved):
> we could create a small workspace class that is smaller than 30Gi disk, for example, 20Gi, since it's for testing purposes.

@jenting I'd prefer the suggestion of using smaller classes and then keeping the storage definition at "30Gi".  We could even go as low as 10gb and 20gb for the workspace classes as I don't expect people to run any workspaces larger than that in preview environments ☺️

I added the hold label and I'm approving so you can merge once you've that change ☺️

jenting commented:
> > we could create a small workspace class that is smaller than 30Gi disk, for example, 20Gi, since it's for testing purposes.
> 
> @jenting I'd prefer the suggestion of using smaller classes and then keeping the storage definition at "30Gi". We could even go as low as 10gb and 20gb for the workspace classes as I don't expect people to run any workspaces larger than that in preview environments ☺️
> 
> I added the hold label and I'm approving so you can merge once you've that change ☺️

Thank you @mads-hartmann 

I change to using a smaller workspace class where the disk size is 20G.
I have updated the code.

/unhold
